verilog/parser/verilog.y

```diff
@@ -5479,6 +5479,18 @@ continuous_assign
     { $$ = MakeTaggedNode(N::kContinuousAssign, $1, $2, $3, $4, nullptr); }
   ;
 
+net_alias_assign
+  : lpvalue '=' net_alias_assign
+    { $$ = ExtendNode($1, $2, $3); }
+  | lpvalue
```

Comment:
Since the grammar for this requires at least two components to be aliased, the base case rule could be a doubleton instead of a singleton:
`| lpvalue '=' lpvalue`

Reply:
Sure, changed